CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **BUG FIXES**
 - Fix inconsistent scaling configuration after cluster update rollback when modifying the list of instance types declared in the Compute Resources.
+- Fix users SSH keys generation when switching users without root privilege in Active Diretory integrated clusters.
```

Comment:
typo: Diretory => Directory.

Moreover, it's better to say with "cluster integrated with external LDAP server" rather than "Active Directory". Active Directory is an implementation of an LDAP server + something else, but pcluster integrates with LDAP server in general, not with AD.

Reply:
Done